aten/src/ATen/native/mkldnn/RegisterMkldnnOpContextClass.cpp

```diff
@@ -53,10 +53,14 @@ TORCH_LIBRARY(mkldnn, m) {
       "mkldnn::_convolution_pointwise_.binary(Tensor(a!) other, Tensor X, Tensor W, Tensor? B, int[] padding, int[] stride, int[] dilation, int groups, str binary_attr, Scalar? alpha, str? unary_attr, Scalar?[] unary_scalars, str? unary_algorithm) -> Tensor(a!) Y"));
   m.def(TORCH_SELECTIVE_SCHEMA(
       "mkldnn::_convolution_transpose_pointwise(Tensor X, Tensor W, Tensor? B, int[] padding, int[] output_padding, int[] stride, int[] dilation, int groups, str attr, Scalar?[] scalars, str? algorithm) -> Tensor Y"));
+  m.def(TORCH_SELECTIVE_SCHEMA(
+      "mkldnn::_lstm(Tensor input, Tensor[] hx, Tensor[] params, bool has_biases, int num_layers, float dropout, bool train, bool bidirectional, bool batch_first) -> (Tensor Y, Tensor hy, Tensor cy)"));
```

Comment:
The existing aten lstm OP is a CompositeImplicitAutograd OP and we're not able to register meta function for it.
We need to replace the original lstm forward call with this `mkldnn::_lstm` OP (in inference case), otherwise, it will be decomposed.

Reply:
The new OP is not needed anymore after moving the weight prepack to be post AOTAutograd and I've removed the related code of the new OP in this PR.